### ISSUE TITLE
Suggestion - Boat Store Implementation at the Marina

### ISSUE THREAD

**Issue by KatLikesTacos** [InZidiuZ/legacy-assets-issue-tracker#432]
**Name:**
KatLikesTacos#6617

**Summary:**
I'd like the below linked YMAP implemented into the world at the Marina down by the airport. The YMAP uses existing vanilla GTA V assets, and thus does not have an associated DLO. This would be in anticipation of any potential work made in relation to https://github.com/InZidiuZ/op-framework-issue-tracker/issues/1714. Now that aircraft have been implemented, I'd love to see boats get some similar treatment, and this YMAP utilizes a currently unused location to fulfill the dealership function of this suggestion.

**Link:**
https://www.gta5-mods.com/maps/boat-dealer-office-ymap

**Reason:**
I've submitted a business proposal via the state IC to open a boat dealership in the city. I feel that being able to legally obtain watercraft would greatly expand the kind of events businesses and state can put on, as boats are not currently legally obtainable. Additionally, having boats that can be used on command allows for events at the yachts to happen far more easily. With the coming implementation of in game licenses, boats could add additional RP to that mechanic, and potentially allow PD to also police on the water as well. With fishing coming in the future as well, people will definitely want to take boats out onto the water to go on fishing trips!

I think that this could expand existing and upcoming RP in the city, and have asked around for awhile and gotten a lot of feedback from other citizens/players that they'd love to see something like this implemented.

**Comment by KrigTroff** [comment]
I agree, this would be a great addition to the city. I would love to own a boat or some kind of water craft.

**Comment by marcf1979** [comment]
would make life easier for those of us who like to go out on the water

**Comment by stubiewonder** [comment]
i would love this. being able to own a boat would be amazing.

**Comment by TheLemonaded** [comment]
This has to happen! Great idea!

**Comment by exclusivegirl** [comment]
This would also be a huge benefit for EMS/Search and Rescue when searching for and assisting drowned patients. Please implement!

**Comment by claymstehling** [comment]
I WOULD LOVE THIS

**Comment by z-Lunar** [comment]
I appreciate the suggestion of things like this, but I suggest aim clear of builds made of Props to save performance 😄

**Comment by KatLikesTacos** [comment]
> I appreciate the suggestion of things like this, but I suggest aim clear of builds made of Props to save performance 😄

Sorry, I'm not 100% sure what you mean. Are you saying this isn't feasible because it's just made of in game assets? Just trying to understand, thanks! 😊

**Comment by DaxterPancake** [comment]
There are two different (probably more) types of maps that are made for FiveM.

One of them is called a YMAP and one of them is called a MLO. A YMAP is someone taking existing props in the game and using those to make a map. This means that a lot of props are being used and loaded, leading to frame loss and general server quality loss. A MLO is an interior that is modelled and made externally using programs like 3DS Max and is a much better resource friendly way of making maps and it is extremely better for quality.

**Comment by z-Lunar** [comment]
> > I appreciate the suggestion of things like this, but I suggest aim clear of builds made of Props to save performance 😄
> 
> Sorry, I'm not 100% sure what you mean. Are you saying this isn't feasible because it's just made of in game assets? Just trying to understand, thanks! 😊

This idea is not out the window 😄.

Daxter also beat me to a reply for YMAPs vs MLO

**Comment by KatLikesTacos** [comment]
> > > I appreciate the suggestion of things like this, but I suggest aim clear of builds made of Props to save performance 😄
> > 
> > 
> > Sorry, I'm not 100% sure what you mean. Are you saying this isn't feasible because it's just made of in game assets? Just trying to understand, thanks! 😊
> 
> This idea is not out the window 😄.
> 
> Daxter also beat me to a reply for YMAPs vs MLO

Gotcha. Should I start investigating alternative options, just to be safe? Or is this not something I need to do work with? Again, sorry, I'm just trying to understand 😅 I don't want to just be throwing ideas/problems at the wall without trying to be part of the solution

**Comment by z-Lunar** [comment]
> > > > I appreciate the suggestion of things like this, but I suggest aim clear of builds made of Props to save performance 😄
> > > 
> > > 
> > > Sorry, I'm not 100% sure what you mean. Are you saying this isn't feasible because it's just made of in game assets? Just trying to understand, thanks! 😊
> > 
> > 
> > This idea is not out the window 😄.
> > Daxter also beat me to a reply for YMAPs vs MLO
> 
> Gotcha. Should I start investigating alternative options, just to be safe? Or is this not something I need to do work with? Again, sorry, I'm just trying to understand 😅 I don't want to just be throwing ideas/problems at the wall without trying to be part of the solution

Lets leave it here so we don't lose it, but if you find others post them 😄

**Comment by riedling11** [comment]
Tossing in my support for this idea!

**Comment by KatLikesTacos** [comment]
So, looping back on this. I haven't really found anything ideal, and I've been looking for other options. If an interior is needed to move forward there's one that could work, it just doesn't really fit with the aesthetics of the rest of the server in my opinion. 🤷‍♀️

**Comment by InZidiuZ** [comment]
This issue tracker is being shut down and replaced by Discord threads. If this issue is still relevant, create a thread on [Discord](https://discord.gg/opfw).
